test/e2e/specs/login.js

```diff
@@ -0,0 +1,71 @@
+const LoginPage = require('../page-objects/login');
+
+describe('Login Page', () => {
+	beforeEach(() => {
+		LoginPage.open();
+	});
+
+	it('should show the header, login box , and all login box elements', () => {
+		expect(LoginPage.header.isVisible()).to.equal(true);
+		expect(LoginPage.loginBox.isVisible()).to.equal(true);
+		expect(LoginPage.email.isVisible()).to.equal(true);
+		expect(LoginPage.password.isVisible()).to.equal(true);
+		expect(LoginPage.login.isVisible()).to.equal(true);
+		expect(LoginPage.forgotPassword.isVisible()).to.equal(true);
+		expect(LoginPage.forgotPassword.getAttribute('href')).to.equal('/reset');
+		expect(LoginPage.register.isVisible()).to.equal(true);
+		expect(LoginPage.register.getAttribute('href')).to.equal('/register');		
+	});
+
+	it('should show an error when no email is provided on a blur event', () => {
+		LoginPage.email.hasFocus() = true;
+		expect(LoginPage.email.getText()).to.equal('');
+		LoginPage.email.hasFocus() = false;
+		expect(LoginPage.emailReqError.isVisible()).to.equal(true);
```

Comment:
Might be good to check that it's not visible before we even focus the email field. This doesn't test that it appears after, just that it shows at some point.